web_auto_refresh/README.rst

```diff
@@ -0,0 +1,78 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Web Auto Refresh
+===========
+
+Standard odoo 10.0 only support auto refreshing the inbox message, for kanban and list view when underlying model data updated by others, even though the auto search option in ir.actions.act_window activated, system will not auto refresh the kanban and list view as expected, this small module just fills this gap, this module is useful for keep monitoring orders, tickets etc without manual refresh the browser.
```

Comment:
what a looong sentence :smile: What about:
"""
Standard odoo 10.0 only support auto refreshing the inbox message.
For kanban and list view when underlying model data have been updated by others, even though the auto search option in ir.actions.act_window activated, the UI won't be refreshed as expected.
This module fills this gap and is useful to keep monitoring orders, tickets and such without manual refresh.
"""
?